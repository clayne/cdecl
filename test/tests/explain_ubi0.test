EXPECTED_EXIT=65
cdecl <<END
explain unsigned _BitInt(0) x
END
