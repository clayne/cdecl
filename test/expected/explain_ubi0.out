explain unsigned _BitInt(0) x
        ^
9: error: unsigned _BitInt must be at least 1 bit
